cmd/antrea-controller/controller.go

```diff
@@ -112,7 +114,8 @@ func run(o *Options) error {
 	anpInformer := crdInformerFactory.Crd().V1alpha1().NetworkPolicies()
 	tierInformer := crdInformerFactory.Crd().V1alpha1().Tiers()
 	tfInformer := crdInformerFactory.Crd().V1alpha1().Traceflows()
-	cgInformer := crdInformerFactory.Crd().V1alpha2().ClusterGroups()
+	cgLegacyCompatibleInformer := crdInformerFactory.Crd().V1alpha2().ClusterGroups()
```

Comment:
maybe name it cgv1a2Informer to avoid confusion as "legacy" is used to represent APIs in "tanzu.vmware.com"?

Reply:
Done